packages/frontend/src/hooks/useETHPrice.ts

```diff
@@ -37,8 +37,7 @@ export const useETHPrice = (refetchIntervalSec = 30) => {
  */
 export const useOnChainETHPrice = () => {
   const index = useAtomValue(indexAtom)
-
-  return toTokenAmount(index, 18).sqrt()
+  return index
```

Comment:
the name `useOnChainETHPrice ` implies it's going to return ethPrice, it's better to return eth price instead of sqrt()